z3/src/tactic.rs

```diff
@@ -103,6 +103,20 @@ impl<'ctx> Tactic<'ctx> {
         }
     }
 
+    /// Return a tactic that applies the current tactic to the goal, failing if
+    /// `t` doesn't terminate in `timeout_ms` milliseconds.
+    pub fn try_for(&self, timeout_ms: u32) -> Tactic<'ctx> {
```

Comment:
Could we use [Duration](https://doc.rust-lang.org/std/time/struct.Duration.html) here?